Source/SourceKittenFramework/Request.swift

```diff
@@ -340,3 +340,70 @@ extension Request: CustomStringConvertible {
     /// A textual representation of `Request`.
     public var description: String { return String(UTF8String: sourcekitd_request_description_copy(sourcekitObject))! }
 }
+
+private func interfaceForModule(module: String) -> [String: SourceKitRepresentable] {
+    let xctestConfigurationPath = NSProcessInfo.processInfo().environment["XCTestConfigurationFilePath"]!
+    let buildPath = xctestConfigurationPath.substringToIndex(xctestConfigurationPath.rangeOfString("/Intermediates/")!.startIndex)
```

Comment:
I got crash at here on testing.
It seems my `xctestConfigrationPath` does not contains `/Intermediates/`.
<img width="1151" alt="screenshot 2016-03-17 14 41 12" src="https://cloud.githubusercontent.com/assets/33430/13837418/6e298d7c-ec4e-11e5-8d66-6a57dea54658.png">